luxurypresence/nestjs-plunk
# [LIFT-990] Add supported node versions and upgrade dependencies

## Issue Description

- Add supported versions
- Upgrade dependencies.

## Issue Tickets
https://linear.app/luxurypresence/issue/LIFT-990/upgrade-nestjs-plunk-node-version-to-20

## Test Plan
- If you want to test with search service remove   `"engines": {
    "node": ">=20.0.0 || >=18.0.0"
  }, ` as SS is not on node 20 yet
- Build the package and use it in SS
- Confirm valuations route still works eg `http://localhost:8010/api/v1/valuations?streetNumber=1900&streetName=Post Street&city=San Francisco&state=CA&zipcode=94115`
